src/renderer/src/components/Header.tsx

```diff
@@ -9,13 +9,14 @@ import MenuIcon from '@mui/icons-material/Menu'
 import BookRoundedIcon from '@mui/icons-material/BookRounded'
 import Menu from '../components/Menu'
 import Layout from '../components/Layout'
-import { useSetRecoilState } from 'recoil'
+import { useRecoilValue, useSetRecoilState } from 'recoil'
 import { ColorSchemeToggle } from '../components/ColorSchemeToggle'
-import { drawerOpenState, screenState } from '../state'
+import { localPeerState, drawerOpenState, screenState } from '../state'
 
 export function Header() {
   const setDrawerOpen = useSetRecoilState(drawerOpenState)
   const setScreen = useSetRecoilState(screenState)
+  const { name } = useRecoilValue(localPeerState)
```

Comment:
Then we can just destructure the name from the atom state here in the component.